template/src/main/twirl/ch.epfl.scala.index.views/project/dependencies.scala.html

```diff
@@ -11,6 +11,13 @@
   }
 </div>
 
+@if(release.internalDependencies.nonEmpty) {
+<div class="dependencies box">
+  <h4>@release.internalDependencyCount Internal Dependencies</h4>
```

Comment:
Some day, we should pluralize these messages so that it's
"1 internal dependency"
" 2 internal dependencies"
This PR is fine because it's consistent with the other boxes.